### PR TITLE
Added Day 6 progress

### PR DESCRIPTION
**Pull request by orrgato**
Completed the pre-requisites , almost. Thinking to skip linux course by edx , It will take too much of time.
I can learn linux more when using it. 
Entering  Advanced stuff tomorrow.

### CODE REVIEW

**Comment by orrgato** [comment]
There are so much video links under each topic . Docket topic has 9 video links .
Is it okay to complete just one video per topic ?

File: Shiwam Verma/day06/day6.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Completed YAML course by kunal kushwaha.
+- Learnt about 
+   - Serialization , DeSerialization
+   - Benefits of YAML
+   - Storing data in YAML
+
+- YAML is the language used in majority in kubenetes, Docker.
+- It's not difficult to write or unnderstand. It's like the next gen of json format.
```
Comment (kranurag7) [inline Shiwam Verma/day06/day6.md:28]:
```suggestion
- YAML is the language which is used in majority in Kubernetes & Docker Compose.
- It's not difficult to write or understand. It's like the next gen of JSON format. 
```

**Review by kranurag7 — CHANGES_REQUESTED** [review]

**Comment by kranurag7** [comment]
@orrgato Go through the suggestions. If you agree then only commit.

**Comment by kranurag7** [comment]
Regarding the doubts, Feel free to ask Saiyam in the discord server. Put you doubts in the `devops-roadmap` channel.

**Pull request merged**